### PR TITLE
Release 0.16.0

### PR DESCRIPTION
**Pull request by 2opremio**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+	[#1576](https://github.com/weaveworks/scope/issues/1576)
+- Container in multiple networks not showing all connections
+	[#1573](https://github.com/weaveworks/scope/issues/1573)
+
+Documentation:
+- Clarify kubectl version matching
+	[#1582](https://github.com/weaveworks/scope/pull/1582)
+
+Internal improvements and cleanup:
+- Add Indentity middleware
```
Comment (rade) [inline CHANGELOG.md]: typo 'Indentity'

**Comment by rade** [comment]
I don't see #1551 mentioned, which a user stumbled across only today.

**Comment by 2opremio** [comment]
@rade I incorporated it later, our CHANGELOG generator script wasn't detecting the new squashed merges properly.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,84 @@
...
+	[#1576](https://github.com/weaveworks/scope/issues/1576)
+- Container in multiple networks not showing all connections
+	[#1573](https://github.com/weaveworks/scope/issues/1573)
+- scope probe connects to localhost & prod even when given explicit hostnames
+	[#1566](https://github.com/weaveworks/scope/issues/1566)
+- Fix Docker for Mac check
+	[#1551](https://github.com/weaveworks/scope/pull/1551)
+- If k8s objects only have one container, show that container's metrics on them
+	[#1473](https://github.com/weaveworks/scope/pull/1473)
+- Don't every store NEW conntrack flows (only every store updates).
```
Comment (rade) [inline CHANGELOG.md]: s/every/ever/g

**Comment by rade** [comment]
About half the items end in a full stop; the other half don't. Is there a rationale?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,84 @@
+## Release 0.16.0
+
+Highlights:
+
+* New network view to quickly ascertain what networks your containers belong to.
```
Comment (rade) [inline CHANGELOG.md]: It's not really a view, is it? @davkal how should we call this new features? "Networks on Canvas"?

**Comment by 2opremio** [comment]
> About 25% of the items end in a full stop; the others don't. Is there a rationale?

Items are auto-generated from PR titles so no rationale. To be honest I don't think it's very important (and it's been inconsistent in previous releases). If we really really want to fix this, we should test PR titles or be strict about them in reviews.

**Comment by rade** [comment]
> To be honest I don't think it's very important

it isn't.

File: CHANGELOG.md (hunk 3 above)
Comment (2opremio) [inline CHANGELOG.md]: Open to any option, I am not too happy with I wrote either.

Comment (rade) [inline CHANGELOG.md]: The ContainerSolutions folks called these things "filters" in a screencast I've just seen. I'd be happy with that.

**Comment by 2opremio** [comment]
@jml @foot @davkal Please test. I need a LGTM before releasing (which I would like to do in less than two hours from now if everything goes well).

**Comment by 2opremio** [comment]
@pidster you too please

**Comment by 2opremio** [comment]
I've tested:
1. The ECS guide
2. Scope monitoring the service when running locally
3. A scope probe attached to cloud.weave.works while running (2)

Found https://github.com/weaveworks/scope/issues/1598 when testing (2) but I don't have evidence of it being part of 0.16 (plus I wouldn't say it's a blocker)

Also, we should upgrade the service to support new app-side features like https://github.com/weaveworks/scope/pull/1597

**Comment by davkal** [comment]
LGTM

**Comment by foot** [comment]
LGTM!

**Comment by pidster** [comment]
LGTM

**Comment by jml** [comment]
I abstain.

**Pull request merged**